### PR TITLE
[WIP] Fix bootstrap (simplified) for macOS.

### PR DESCRIPTION
**Pull request by luenix**
Simplified PR (in contrast to #193) with intent to fix bootstrap for macOS. #

### CODE REVIEW

File: packages/transmute-cli/src/commands/provision/index.js [hunk 1]
```diff
@@ -1,107 +1,119 @@
 const run = require('../runner');
 const logger = require('../../logger');
-const TRANSMUTE_KUBE_VERSION = process.env.TRANSMUTE_KUBE_VERSION || 'v1.9.0';
+
```
Comment (luenix) [inline packages/transmute-cli/src/commands/provision/index.js]: Whitespace fix.

File: packages/transmute-cli/src/commands/provision/index.js [hunk 2]
```diff
@@ -1,107 +1,119 @@
...
+const TRANSMUTE_KUBE_VERSION = process.env.TRANSMUTE_KUBE_VERSION || 'v1.10.0';
```
Comment (luenix) [inline packages/transmute-cli/src/commands/provision/index.js]: Change to Kubernetes 1.10.0.  Kubernetes 1.9.4 was causing a failure to schedule minikube addons "storage-provisioner" and "dashboard" on macOS OS X 10.13.

File: packages/transmute-cli/src/commands/provision/index.js [hunk 3]
```diff
@@ -1,107 +1,119 @@
...
-        ' -e kubernetes_version=' +
-        TRANSMUTE_KUBE_VERSION +
-        ' -e minikube_disk_size=' +
-        MINIKUBE_DISK +
-        ' -e minikube_cpus=' +
-        MINIKUBE_CPU +
-        ' -e minikube_memory=' +
-        MINIKUBE_MEMORY;
-    +' -e minikube_profile=' + MINIKUBE_PROFILE;
+module.exports.minikube = (dryrun, minikubeDriver) => {
```
Comment (luenix) [inline packages/transmute-cli/src/commands/provision/index.js]: From here until end of file, commit fixes indentation (4 spaces corrected to 2 spaces).

File: ansible-playbook/transmute.yml [hunk 4]
```diff
@@ -2,7 +2,7 @@
 - name: transmute
   vars:
     fix_dotfiles: true
-    transmute_kube_version: "v1.9.4"
+    transmute_kube_version: "v1.10.0"
```
Comment (luenix) [inline ansible-playbook/transmute.yml]: Change to Kubernetes 1.10.0. Kubernetes 1.9.4 was causing a failure to schedule minikube addons "storage-provisioner" and "dashboard" on macOS OS X 10.13.

File: ansible-playbook/roles/docker/tasks/main.yml [hunk 5]
```diff
@@ -33,21 +33,22 @@
...
   when: ansible_system == "Darwin"
 
-- name: start docker
+- name: Start docker
   service: name=docker state=started enabled=yes
   become: yes
   when: ansible_system == "Linux"
 
-- name: Install docker
+- name: Install docker (pip)
```
Comment (luenix) [inline ansible-playbook/roles/docker/tasks/main.yml]: Change task name to be more explicit.

File: ansible-playbook/roles/docker/tasks/main.yml [hunk 6]
```diff
@@ -33,21 +33,22 @@
...
+    name: "docker"
+    state: present
   when: ansible_system == "Darwin"
 
 - name: Start docker (Darwin)
   shell: open --background -a Docker
   when: ansible_system == "Darwin"
 
-- name: start docker
+- name: Start docker
```
Comment (luenix) [inline ansible-playbook/roles/docker/tasks/main.yml]: Capitalize task name.

**Comment by OR13** [comment]
try pinning the version in travis.yml and get this passing so I can review it.

**Pull request closed without merging**